docs(security): document log_connections=off default and re-enable path (#47199)
supabase/supabase: apps/docs/content/guides/platform/postgres-connection-logging.mdx

## Compliance considerations

<Admonition type="note">

If you need connection audit evidence for SOC 2 or other compliance programs, you must enable it explicitly.

</Admonition>

Connection logging supports audit and monitoring controls required by some compliance programs:

- **HIPAA** — High-compliance projects should keep connection logging enabled. See the [shared responsibility model for healthcare data](/docs/guides/deployment/shared-responsibility-model#managing-healthcare-data) and [HIPAA compliance guide](/docs/guides/security/hipaa-compliance).
- **SOC 2** — Users who need connection audit evidence should enable logging and retain logs according to their own policies. See the [SOC 2 compliance guide](/docs/guides/security/soc-2-compliance).

Disabling connection logging does not affect other Supabase logging (for example, [Platform Audit Logs](/docs/guides/security/platform-audit-logs), [Auth Audit Logs](/docs/guides/auth/audit-logs), or [pgAudit](/docs/guides/telemetry/logs#configuring-pgauditlog)).

## Manage connection logging via the dashboard

> Connection logging can be configured from the **Log connections** setting in the [Database Settings](/dashboard/project/_/database/settings) page of the dashboard.

Reviewer: ```suggestion
You can configure connection logging  from the **Log connections** setting in the [Database Settings](/dashboard/project/_/database/settings) section of the Dashboard.
```
